### ISSUE TITLE
Problem with installing Adminer

### ISSUE THREAD

**Issue by kusmierz** [puphpet/puphpet#1260]
```
==> default: Error: Cannot create /var/www/default/adminer; parent directory /var/www/default does not exist
==> default: Error: /Stage[main]/Puphpet::Adminer/File[/var/www/default/adminer]/ensure: change from absent to directory failed: Cannot create /var/www/default/adminer; parent directory /var/www/default does not exist
```

Shouldn't be there /var/www/hml/?

see my config in #1258

**Comment by kusmierz** [comment]
Still no luck. I destroyed my machine and `vagrant up` it again... 

```bash
==> default: Error: Cannot create /var/www/default/adminer; parent directory /var/www/default does not exist
==> default: Error: /Stage[main]/Puphpet::Adminer/File[/var/www/default/adminer]/ensure: change from absent to directory failed: Cannot create /var/www/default/adminer; parent directory /var/www/default does not exist
```

could you help me?

```yaml
---
vagrantfile-local:
    vm:
        box: puphpet/ubuntu1404-x64
        box_url: puphpet/ubuntu1404-x64
        hostname: foobar.dev
        memory: '2048'
        cpus: '2'
        chosen_provider: virtualbox
        network:
            private_network: 192.168.56.101
            forwarded_port:
                RWrkfZZPI0EN:
                    host: '4040'
                    guest: '4040'
                E2064DsOT93X:
                    host: '8288'
                    guest: '22'
                fWZExnytqf0e:
                    host: '8080'
                    guest: '8080'
        post_up_message: ''
        provider:
            virtualbox:
                modifyvm:
                    natdnshostresolver1: on
            vmware:
                numvcpus: 1
            parallels:
                cpus: 1
        provision:
            puppet:
                manifests_path: puphpet/puppet
                manifest_file: site.pp
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
                    - '--parser future'
        synced_folder:
            KFtZN0L1yIbl:
                owner: ''
                group: ''
                source: ./
                target: /var/www/foobar/
                sync_type: nfs
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .git/
                        - .vagrant/
                    auto: 'false'
        usable_port_range:
            start: 10200
            stop: 10500
    ssh:
        host: null
        port: null
        private_key_path: null
        username: vagrant
        guest_port: null
        keep_alive: true
        forward_agent: false
        forward_x11: false
        shell: 'bash -l'
    vagrant:
        host: detect
server:
    install: '1'
    packages:
        - vim
        - cron
        - anacron
        - curl
        - fluxbox
        - grep
        - gnupg
        - imagemagick
        - imagemagick-common
        - xvfb
        - phantomjs
        - git
        - git-core
        - git-flow
        - screen
        - htop
        - bmon
        - atop
        - iotop
        - lftp
        - libpcre3
        - libpcre3-dev
        - libssl-dev
        - zlib1g-dev
        - wget
        - tar
        - gzip
        - unrar
        - rar
        - bash-completion
        - ethstatus
        - ifstat
        - iftop
        - iptraf
        - host
        - links2
        - libdate-manip-perl
        - locate
        - sendmail
        - mailutils
        - mutt
        - util-linux
        - at
        - translate-toolkit
        - httpie
users_groups:
    install: '1'
    groups: {  }
    users: {  }
cron:
    install: '1'
    jobs:
        5uux7mh5oha4:
            monthday: ''
            name: composer-update
            user: root
            command: '/usr/bin/php /usr/local/bin/composer selfupdate > /dev/null 2>&1'
            minute: '0'
            hour: '*'
            weekday: '*'
            month: '*'
        jfctx0g48rkb:
            monthday: ''
            name: foobar-cron
            user: www-data
            command: 'cd /var/www/foobar/src/head/ && /usr/local/bin/php cron.php'
            environment:
                - APPLICATION_ENV=vagrant
            minute: '*'
            hour: '*'
            weekday: '*'
            month: '*'
firewall:
    install: '1'
    rules: null
apache:
    install: '0'
    settings:
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - rewrite
    vhosts:
        f4ogatvbjvcl:
            servername: awesome.dev
            serveraliases:
                - www.awesome.dev
            docroot: /var/www/awesome
            port: '80'
            setenv:
                - 'APPLICATION_ENV vagrant'
            directories:
                gwanacfrtjym:
                    provider: directory
                    path: /var/www/awesome
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - all
                        - granted
                    custom_fragment: ''
            engine: php
            custom_fragment: ''
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
    mod_pagespeed: 0
nginx:
    install: '1'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
    upstreams: {  }
    vhosts:
        t1tma2dhaxaz:
            proxy: ''
            server_name: admin.foobar.dev
            server_aliases:
                - .admin.foobar.dev
            www_root: /var/www/foobar/public-admin/
            listen_port: '80'
            location: .php$
            index_files:
                - index.php
            envvars:
                - 'APPLICATION_ENV vagrant'
            engine: php
            client_max_body_size: 1m
            ssl_cert: ''
            ssl_key: ''
        ir9etrsksfru:
            proxy: ''
            server_name: api.foobar.dev
            www_root: /var/www/foobar/public-api/
            listen_port: '80'
            location: .php$
            index_files:
                - index.php
            envvars:
                - 'APPLICATION_ENV vagrant'
            engine: php
            client_max_body_size: 1m
            ssl_cert: ''
            ssl_key: ''
        6ae3sbfvvrjw:
            proxy: ''
            server_name: foobar.dev
            server_aliases:
                - foobar.test.com
                - .foobar.test.com
                - www.foobar.dev
            www_root: /var/www/foobar/public/
            listen_port: '80'
            location: .php$
            index_files:
                - index.php
                - index.html
                - index.htm
            envvars:
                - 'APPLICATION_ENV vagrant'
            engine: php
            client_max_body_size: 1m
            ssl_cert: ''
            ssl_key: ''
php:
    install: '1'
    version: '56'
    composer: '1'
    composer_home: ''
    modules:
        php:
            - cli
            - common
            - curl
            - gd
            - gmp
            - imagick
            - intl
            - mbstring
            - mcrypt
            - memcached
            - sqlite
        pear: {  }
        pecl:
            - PDO
            - PDO_PGSQL
            - redis
            - date_time
            - timezonedb
            - gnupg
            - Fileinfo
            - zip
            - pecl_http
            - intl
            - big_int
            - uuid
            - igbinary
    ini:
        display_errors: On
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        short_open_tag: Off
        expose_php: Off
        max_execution_time: '300'
        max_input_time: '300'
        memory_limit: 1G
        display_startup_errors: On
        log_errors: On
        log_errors_max_len: '1024'
        error_log: /var/log/php/error.log
        ignore_repeated_errors: Off
        ignore_repeated_source: Off
        report_memleaks: On
        post_max_size: 32M
        upload_max_filesize: 32M
        cgi.fix_pathinfo: '0'
        filter.default: unsafe_raw
        default_charset: utf-8
        realpath_cache_size: 1024K
        opcache.enable: '1'
        opcache.memory_consumption: '255'
        opcache.max_accelerated_files: '16384'
        opcache.revalidate_freq: '0'
        opcache.use_cwd: '1'
        opcache.validate_timestamps: '1'
        opcache.save_comments: '1'
        opcache.enable_file_override: '0'
        opcache.enable_cli: '1'
        opcache.file_update_protection: '0'
    timezone: Europe/Warsaw
    mod_php: 0
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings:
        host: 127.0.0.1
        port: '9000'
    ini: {  }
    timezone: null
xdebug:
    install: '1'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_host: 192.168.56.1
        xdebug.remote_enable: '1'
        xdebug.remote_mode: req
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
        xdebug.scream: '0'
        xdebug.max_nesting_level: '250'
        xdebug.cli_color: '1'
        xdebug.profiler_enable: '0'
        xdebug.profiler_enable_trigger: '1'
        xdebug.profiler_output_name: profiler.out.%t.%p
        xdebug.trace_format: '1'
        xdebug.collect_assignments: '1'
        xdebug.collect_params: '3'
        xdebug.collect_return: '1'
        xdebug.collect_vars: '1'
        xdebug.show_local_vars: '0'
        xdebug.show_exception_trace: '0'
        xdebug.var_display_max_depth: '5'
        xdebug.var_display_max_children: '256'
        xdebug.var_display_max_data: '1024'
xhprof:
    install: '0'
wpcli:
    install: '0'
    version: ''
drush:
    install: '0'
    version: 6.3.0
ruby:
    install: '1'
    versions:
        IB0ucuMG52Tu:
            default: '1'
            version: 1.9.3
            gems:
                - bourbon
                - compass
                - sass
            bundler: '1'
nodejs:
    install: '1'
    npm_packages:
        - bower
        - test
        - requirejs
        - grunt
        - grunt-cli
        - yo
        - generator-composer
        - generator-thorax
python:
    install: '1'
    packages: {  }
    versions: {  }
mysql:
    install: '0'
    override_options: {  }
    root_password: '123'
    adminer: 0
    databases:
        DtMQea1br3cP:
            grant:
                - ALL
            name: dbname
            host: localhost
            user: dbuser
            password: '123'
            sql_file: ''
postgresql:
    install: '1'
    settings:
        root_password: '123'
        user_group: postgres
        encoding: UTF8
        version: '9.3'
    databases:
        c1pxY0TeEwaB:
            grant: ALL
            name: foobar
            user: foobar
            password: foobar
            sql_file: ''
    adminer: '1'
mariadb:
    install: '0'
    override_options: {  }
    root_password: '123'
    adminer: 0
    databases: {  }
    version: '10.0'
sqlite:
    install: '0'
    adminer: '0'
    databases: {  }
mongodb:
    install: '0'
    settings:
        auth: 1
        port: '27017'
    databases: {  }
redis:
    install: '1'
    settings:
        conf_port: '6379'
mailcatcher:
    install: '1'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '1080'
        mailcatcher_path: /usr/local/rvm/wrappers/default
        from_email_method: inline
beanstalkd:
    install: '1'
    settings:
        listenaddress: 0.0.0.0
        listenport: '13000'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: '1'
    binlogdir: /var/lib/beanstalkd/binlog
rabbitmq:
    install: '0'
    settings:
        port: '5672'
elastic_search:
    install: '0'
    settings:
        version: ''
        java_install: true
solr:
    install: '0'
    settings:
        version: ''
        port: ''
```

**Comment by jtreminio** [comment]
Would you mind generating your archive again? I fixed it in commit 5b64cf6d351cdcfa2b8ca8dd583a6e78ea3c78b7

**Comment by kusmierz** [comment]
Thanks, I'll try!
